commercetools-models/src/test/java/io/sphere/sdk/models/AddressTest.java

```diff
@@ -5,7 +5,7 @@
 
 import static org.assertj.core.api.Assertions.*;
 
-public final class AddressTest {
+public class AddressTest {
```

Comment:
Actually the subclassing plugin, needs the tests to be not final in order to be able to create subclasses without compile error, that's the main reason why i made so many classes not final, and that's also the reason why at least one of their constructors need to be non private